sdk/communication/Azure.Communication.Email/src/EmailClient.cs

```diff
@@ -22,7 +20,7 @@ public partial class EmailClient
     {
         private readonly ClientDiagnostics _clientDiagnostics;
 
-        internal EmailRestClient RestClient { get; }
+        private readonly EmailRestClient RestClient;
```

Comment:
nit: field naming convention would be `_restClient`

Reply:
updated.